add definition for uchar type that is not known by Intel compilers
author: Kenneth Hoste (HPC-UGent)
--- gdal-2.2.0/frmts/jpeg2000/jpeg2000_vsil_io.h.orig	2017-05-15 17:23:31.334324922 +0200
+++ gdal-2.2.0/frmts/jpeg2000/jpeg2000_vsil_io.h	2017-05-15 17:23:52.134563207 +0200
@@ -34,4 +34,6 @@
 
 jas_stream_t *JPEG2000_VSIL_fopen(const char *filename, const char *mode);
 
+#define uchar unsigned char
+
 #endif
